src/SolverInterface/lpqp_to_conic.jl

```diff
@@ -7,6 +7,7 @@
 
 type LPQPtoConicBridge <: AbstractConicModel
     lpqpmodel::AbstractLinearQuadraticModel
+    qcp::Bool
```

Comment:
Why do we need a new flag if it's always equal to ``model.Asoc !== nothing || model.Arsoc !== nothing``?